components/nexus-bootstrap/src/main/java/org/sonatype/nexus/bootstrap/jetty/JettyServer.java

```diff
@@ -204,9 +205,16 @@ private void doStop() throws Exception {
     log.info("Stopped");
   }
 
+  // FIXME remove once adding docker connectors properly implemented
+  public static Server getServer() throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00112)